ci-operator/jobs/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main-presubmits.yaml

```diff
@@ -136,6 +136,86 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )rosa-sts-advanced-critical-high-presubmit,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build03
+    context: ci/prow/rosa-sts-advanced-optional-critical-high-presubmit
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws-terraform-qe
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-advanced-optional-critical-high-presubmit
```

Comment:
I'm afraid the name is too long :-( 
Is there a way to improve this?

Reply:
I know that I requested it to be more explicit
Just adding here another thought, need to find the middle :-(

Reply:
the thing is, we will need to change the other jobs to be aligned with this "imperative" term,
maybe not so convenient...

Reply:
Ok, never mind, running the tests with shorter name